python/lsst/dbb/buffmngrs/endpoint/finder/finder.py

```diff
@@ -167,6 +168,7 @@ def run(self):
                     relpath=os.path.relpath(dirname, start=self.storage),
                     filename=basename,
                     checksum=checksum,
+                    size_bytes=status.st_size,
```

Comment:
Same comment as in backfill about better to have a function for the db operation to allow easier additions of more columns.